pkg/vpcmodel/grouping.go

```diff
@@ -86,6 +87,13 @@ type GroupedConnLine struct {
 func (g *GroupedConnLine) String() string {
 	return g.Src.Name() + " => " + g.Dst.Name() + " : " + g.Conn
 }
+func (g *GroupedConnLine) ConnLabel() string {
+	// todo - this info can be found in the conn struct.
+	if strings.Contains(g.Conn, "All Connections") {
+		return ""
+	}
+	return g.Conn
```

Comment:
can be added as a function exported by`common/connectionset.go` that checks if a connection string represents all connections or not. (so that won't need to duplicate the hard-coded str "All Connections").

Reply:
done